### PR TITLE
feat: edit feature

### PR DESCRIPTION
**Pull request by devsurajrai**
added note edit feature. please review and provide your valuable comments.

LIVE: https://deploy-preview-7--note-phoenix.netlify.app/

### CODE REVIEW

File: src/util/addNote/addNote.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import { createNote } from "../../backendUtil/backendUtil";
 
-export const addNote = (note, setIsCreateNewNote, userToken, authDispatch) => {
+export const addNote = (
```
Comment (SJTGSHIVAM) [inline src/util/addNote/addNote.js:3]: I like the way you changed it to multiple line as discussed in previous PR.

**Review by SJTGSHIVAM — APPROVED** [review]
The code looks clean and good to merge.

File: src/util/editNote/editNote.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  //   let noteToUpdate = {};
+  //   for (let note of notes) {
+  //     if (note._id === noteId)
+  //       noteToUpdate = { ...note, isPinned: !note.isPinned };
+  //   }
+  //   (async () => {
+  //     const updatedNote = await updateNote(noteToUpdate, userToken, noteId);
+  //     authDispatch({ type: "SET_NOTES", payload: { value: updatedNote } });
+  //   })();
```
Comment (Dev-AkshatGupta) [inline src/util/editNote/editNote.js:32]: remove the commented code

**Review by Dev-AkshatGupta — APPROVED** [review]
One small suggestion for proper indentation 
Else everything seems good too me

**Pull request merged**